src/design_patterns/ShipmentKinds.ts

```diff
@@ -0,0 +1,89 @@
+import {AirEastShipper, ChicagoSprintShipper, PacificParcelShipper} from './Shipper'
+
+export interface IShipmentKinds {
+    getAirEastShipperRate: (w?: number) => number
```

Comment:
`w` - it's a bad name. Try to use understandable for readers naming.

Reply:
fixed